### PR TITLE
ACCEPTANCE: EFxceptionsContext

### LINKED ISSUE #10: ACCEPTANCE:  EFxceptionsContext Acceptance Tests

### PR DESCRIPTION
**Pull request by ZafarUrakov**
Closes #10 
@cjdutoit

### CODE REVIEW

File: STX.EFxceptions.Cosmos/STX.EFxceptions.Cosmos.csproj [hunk 1]
```diff
@@ -46,7 +46,7 @@
 
 	<ItemGroup>
 	  <PackageReference Include="Microsoft.Azure.Cosmos" Version="3.37.1" />
-	  <PackageReference Include="STX.EFxceptions.Core" Version="0.1.0" />
+	  <PackageReference Include="STX.EFxceptions.Core" Version="0.1.1" />
```
Comment (cjdutoit) [inline STX.EFxceptions.Cosmos/STX.EFxceptions.Cosmos.csproj]:
```suggestion
	  <PackageReference Include="STX.EFxceptions.Core" Version="0.1.2" />
```

**Comment by glhays** [comment]
@cjdutoit @ZafarUrakov These appear to be Unit tests not Acceptance tests. Unit testing typically fall within Foundations category.

**Comment by ZafarUrakov** [comment]
@glhays The PR(branch) marked as draft may not be ready for review.

**Comment by glhays** [comment]
> @glhays The PR(branch) marked as draft may not be ready for review.

Very well aware of that. Comment isn't a review of the committed work. An observation of the Standard category being applied based on the work being done.
Further more, I am not doing reviews of your work and have not been asked to do so.

**Review by cjdutoit — APPROVED** [review]

**Pull request merged**